src/backend/gl/src/command.rs

```diff
@@ -420,6 +424,10 @@ impl CommandBuffer {
 
             match vertex_buffer_descs.get(binding) {
                 Some(&Some(desc)) => {
+                    if let pso::VertexInputRate::Instance(count) = desc.rate {
+                        attribute.offset += desc.stride * first_instance / count as u32;
```

Comment:
afaik first instance is independent of the instance divisor

Reply:
There's no such thing in Vulkan.
So, what behavior gfx should have?

Reply:
It should behave the same way as defined the vk divisor extension:
```c
if (divisor == 0)
    vertexOffset = firstInstance * bindingDesc.stride;
else
    vertexOffset = (firstInstance + ((instanceIndex - firstInstance) / divisor)) *
bindingDesc.stride;
```